jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -1233,4 +1234,82 @@ public interface Mount extends Closeable
          */
         Resource root() throws IOException;
     }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param resources multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources.
+     * @see ResourceCollection
+     */
+    public static ResourceCollection of(List<Resource> resources)
+    {
+        if (resources == null)
+            throw new IllegalArgumentException("No resources");
+
+        return new ResourceCollection(resources);
+    }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param resources multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources.
+     * @see ResourceCollection
+     */
+    public static ResourceCollection of(Resource... resources)
+    {
+        if (resources == null)
+            throw new IllegalArgumentException("No resources");
+
+        return new ResourceCollection(Arrays.asList(resources));
+    }
+
+    /**
+     * <p>Make a Resource containing a collection of other resources</p>
+     * @param resourceAsStrings multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources or null if no resources passed.
+     * @see ResourceCollection
+     * @throws IllegalArgumentException if a string cannot be converted to resource
+     */
+    public static ResourceCollection of(String... resourceAsStrings)
```

Comment:
Input here can be path references or uri string references.
Path references can be absolute, or relative.
Path references can also be presented in windows or unix formats.
Path references from classpath/classloader style configurations can even include globs like `C:\other\libs\*` or `/opt/web/libs/*`.

It would make more sense to convert them all to URIs without the Resource layer involved first, then you know if there's mounting needing to be done.

Reply:
I'm just going to keep using the existing `fromList` for now.

Reply:
These concerns (about going thru Resource.newResource() during glob calculation has been addressed in the original ResourceCollection PR